ios/RN/RNCamera.m

```diff
@@ -606,7 +606,7 @@ - (void)startSession
         }
 
         // Default video quality AVCaptureSessionPresetHigh if non is provided
-        AVCaptureSessionPreset preset = ([self defaultVideoQuality]) ? [RNCameraUtils captureSessionPresetForVideoResolution:[[self defaultVideoQuality] integerValue]] : AVCaptureSessionPresetHigh;
```

Comment:
Please revert this line, it causes an instant app crash when using the `defaultVideoQuality` property after you changed this line.

Reply:
So is the other method the appropriate way to  use `defaultVideoQuality`? They need to be consistent, right? It definitely fixed the issue for me when stopping recording the first time.